### DISCUSSION TITLE
FLSUN QQ-S Pro Extruder Speed (Flow Rate)

### DISCUSSION THREAD

**Discussion by afp316** [blackfyre/flsun.community#19] [Q&A]
I've found that my print are looking under-extruder, especially in the infill sections.  I've tried slicing models with an extrusion multiplier of 1.1 in the slicing software (PrusaSlicer), however, after I load it to the machine via an SD card, I check the extruder speed (aka flow rate) by going to "Options" - "More" - "Speed" and it continually shows 95%.  Has anyone experienced this before?  MY next course of action will be to replace the existing heat throat with an all metal one and verify it's not clogging but I don't think it's going to be that simple.

**Comment by blackfyre** [comment]
Why not calibrate the e-steps first?
